src/logger/src/logger.rs

```diff
@@ -254,6 +255,13 @@ impl Logger {
             prefix.push(instance_id.to_string());
         }
 
+        // Attach current thread name to prefix.
+        let handle = thread::current();
+        match handle.name() {
+            Some(x) => prefix.push(x.to_string()),
+            None => prefix.push("-".to_string()),
```

Comment:
I am thinking about the decision of setting the default thread name to "-". Citing the Rust documentation, ["By default, spawned threads are unnamed"](https://doc.rust-lang.org/std/thread/#naming-threads), it could mean that we could very well ignore it if no thread name is provided. @sandreim may I have your input on this?